advocacy_docs/migrating/oracle/edb_migration_tools.mdx

```diff
@@ -76,9 +76,9 @@ When reviewing the table, the following apply:
 | Complete cutover/go live              | - **Replication Server** <br/> - **LiveCompare** <br/> - *Embedded Postgres SME Service* |
 
 
-[^1]: PEM can monitor a BigAnimal cluster but with some limitations. The monitoring services provided by the cloud service provider are used in conjunction with PEM or instead of it. PEM isn't applicable for monitoring Cloud Native Postgres instances.  
-[^2]: Failover Manager isn't applicable for BigAnimal or Cloud Native PostgreSQL (CNP) deployments as the capabilities it provides are built natively into the BigAnimal service or the CNP operator.  
-[^3]: EDB Postgres Distributed isn't currently available for the BigAnimal and Cloud Native PostgreSQL offerings. When it becomes available, its capabilities will be integrated as an option directly in the BigAnimal service and Cloud Native Postgres operators.  
+[^1]: PEM can monitor a cluster managed with EDB Postgres AI Cloud Services (formerly BigAnimal) but with some limitations. The monitoring services provided by the cloud service provider are used in conjunction with PEM or instead of it. PEM isn't applicable for monitoring Cloud Native Postgres instances.  
```

Comment:
In markdown, it looks like this is the first occurrence of EDB Postgres AI Cloud Services and, hence, should have a ® symbol. However, the [^1] denotes a footnote, which means that these three points go at the bottom of the page.